Makefile

```diff
@@ -44,38 +43,38 @@ version:
 	@echo Go version: $(shell go version)
 
 install:
-	@(set -e;for i in $(SUBDIRS); do \
-		echo Installing $$i ...; \
-		$(MAKE) -C $$i install; \
-	done)
+	@echo "Installing dgraph..."
+	@$(MAKE) -C dgraph install
 
 install_oss oss_install:
 	$(MAKE) BUILD_TAGS=oss install
 
 uninstall:
-	@(set -e;for i in $(SUBDIRS); do \
-		echo Uninstalling $$i ...; \
-		$(MAKE) -C $$i uninstall; \
-	done)
+	@echo "Uninstalling dgraph..."
+	@$(MAKE) -C dgraph uninstall
 
+# ./test.sh is defunct
+# use t.go in t directory
```

Comment:
add a `TODO` at the start of the comments in one 56-57

Reply:
Done